### PR TITLE
Redesign redirect page

### PR DESCRIPTION
**Pull request by kanchanagodage**
Redesigned and implemented redirect page

**Before**
92477391-256b5600-f1fe-11ea-9c58-5315a116af7d

**After**
Screenshot (494)

Fixes [Issue 9395](https://github.com/wso2/product-is/issues/9395)

### CODE REVIEW

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+</head>
+
+<body onload="javascript:document.forms[0].submit()">
+
+    <div class="padding-top-20">
+        <div class="ui centered grid">
+            <div class="twelve wide mobile eight wide tablet six wide computer column">
+                <div class="redirect-loader">
+                    <i class="fa fa-circle-o-notch fa-spin loader-redirect"></i>
+                    Please wait while you're being redirected...
```
Comment (mefarazath) [inline features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html:76]: We are doing an automatic form post here. So "you're being redirected" might give a wrong impression that we are doing a redirect.

@ayshsandu @janakamarasena WDYT?

Comment (ayshsandu) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html:76]: What about "Please wait while we are processing your request" ?

Comment (ayshsandu) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html:76]: Need to change the next message also accordingly. `Click the submit button if automatic redirection failed.` -> `Click the submit button if automatic submission failed.` ?

Comment (mefarazath) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html:76]: > What about "Please wait while we are processing your request" ?

How about
"Please wait while we complete your request"

Comment (kanchanagodage) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html]: Fixed with fc4b8cf

**Review by DimalChandrasiri — APPROVED** [review]

File: features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html (hunk 1 above)
Comment (DimalChandrasiri) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html]: We changed the phrases to the following after a discussion with @ayshsandu 

Main message
`Please wait while we take you to the application.`

The description for action button 
`Click continue if you have been waiting for too long.`

@mefarazath WDYT of that approach?

Comment (mefarazath) [reply features/org.wso2.carbon.identity.oauth.server.feature/resources/oauth_response.html]: +1

**Pull request merged**